python-lib/spell_checker/language_detection.py

```diff
@@ -0,0 +1,19 @@
+import cld3
+
+
+def get_language_of_text(txt):
```

Comment:
to be replaced by spacy-fastlang once we have the validation from Clement on the license